srcpkgs/default-shader-pack/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'default-shader-pack'
+pkgname=default-shader-pack
+version=2.0.0
+revision=1
+build_style=fetch
+hostmakedepends="tar"
+short_desc="MPV shaders and configurations for MPV Shim media clients"
+maintainer="Neel Chotai <neel@chot.ai>"
+license="MIT, Unlicense, LGPL-3.0-or-later, LGPL-3.0-only"
+homepage="https://github.com/iwalton3/default-shader-pack"
+distfiles="https://github.com/iwalton3/default-shader-pack/archive/v${version}.tar.gz
+ https://raw.githubusercontent.com/igv/FSRCNN-TensorFlow/master/LICENSE>LICENSE-FSRCNN
+ https://raw.githubusercontent.com/bloc97/Anime4K/master/LICENSE>LICENSE-Anime4K"
+checksum="765e80bf97d46e00d6f273dbabe9111826f6711c4e60ee786a4c6bcb2299a890
+ 3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986
+ 5bad448b737378e3d0c977ad0d0521fa37ad279a7e76ea9a31d9257eeb6953f5"
+
+do_extract() {
```

Comment:
No need for a custom `do_extract`, just set `skip_extraction="LICENSE-Anime4K LICENSE-FSRCNN"` on the top level and move `vlicense` to `do_install`.

(And remove `build_style` and `hostmakedepends`)